sink/src/main/kotlin/org/neo4j/connectors/kafka/sink/strategy/RelationshipPatternHandler.kt

```diff
@@ -121,6 +123,33 @@ class RelationshipPatternHandler(
         .toList()
   }
 
+  fun validate(constraints: List<ConstraintData>) {
+    val warningMessages = checkConstraints(constraints)
+    warningMessages.forEach { logger.warn(it) }
+  }
+
+  override fun checkConstraints(constraints: List<ConstraintData>): List<String> {
+    val warningMessages = mutableListOf<String>()
+
+    val startNodeWarning =
```

Comment:
singular in the variable name, plural in the method name, that's a bit confusing to me
does this return a single warning or a list thereof?
I cannot tell without looking at the implementation of `checkNodeWarnings`

Reply:
yeah you're right, method used to return a list this is why it was plural but in this case it's good to change them to singular